cmd/ocm-backplane/cloud/common.go

```diff
@@ -333,7 +340,10 @@ func (cfg *QueryConfig) getIsolatedCredentials(ocmToken string) (aws.Credentials
 						Arn: aws.String(roleChainResponse.SessionPolicyArn),
 					},
 				}
+			} else {
+				roleArnSession.Policy = &inlinePolicy
```

Comment:
ok, so the inline session policy only applies to `CustomerRoleArnName` and for the other roles just remains `nil`. Then, it returns to  call `AssumeRoleSequence()` to assume CustomerRole (with the inline policy).

Reply:
Yeah exactly..